packages/mui-base/src/FormControl/FormControl.types.ts

```diff
@@ -115,4 +115,4 @@ export type FormControlRootSlotProps = {
   ownerState: FormControlOwnerState;
 };
 
-export interface UseFormControlContextReturnValue extends FormControlState {}
+export type UseFormControlContextReturnValue = FormControlState;
```

Comment:
Is this change just to test if your change works properly? If so, you could add a unit test to HookApiBuilder

Reply:
Ah, no, not changed for testing. I actually had written it as `type` before for return value docs, but then docs had not been generated, so I wrote it as a `interface`. Just that I did not like empty interface (not really empty because of`extends`). Just a convention I preferred here. 

Do you prefer interface only so that in future properties can be simply be added instead of type intersection? I can revert it.

Reply:
All the other return values are interfaces, so let's keep it consistent where we can.

Reply:
Alright, done.